osu.Game/Storyboards/Drawables/DrawableStoryboardAnimation.cs

```diff
@@ -118,8 +118,24 @@ private void load(TextureStore textureStore, Storyboard storyboard)
             for (int frame = 0; frame < Animation.FrameCount; frame++)
             {
                 string framePath = Animation.Path.Replace(".", frame + ".");
+                Drawable sprite = storyboard.CreateSpriteFromResourcePath(framePath, textureStore);
 
-                AddFrame(storyboard.CreateSpriteFromResourcePath(framePath, textureStore), Animation.FrameDelay);
+                if (sprite != null)
+                {
+                    AddFrame(sprite, Animation.FrameDelay);
+                    continue;
+                }
+
+                framePath = Animation.Path.Replace("0.", frame + ".");
+                sprite = storyboard.CreateSpriteFromResourcePath(framePath, textureStore);
+
+                if (sprite != null)
+                {
+                    AddFrame(sprite, Animation.FrameDelay);
+                }
+
+                // todo: handle animation intentionally declared with more frames than sprites to cause a blinking effect
+                // e.g. beatmap 5381's "spr\play-skip.png"
```

Comment:
And the blinking effect will likely be its own issue

Reply:
could probably be handled by making the animation a `DrawableAnimation` and adding empty drawables (or maybe even empty sprites) as frames